package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "freddie-system-requirements",
+  "version": "1.0.0",
+  "description": "My second module for FEC",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
It makes sense to add some tests to this code. There's a fair bit of react in here that can be tested.

Reply:
Yes I'm running behind on the testing for this service as well as the css but will get to those asap.